app_feup/lib/view/Widgets/HomePageBackButton.dart

```diff
@@ -20,12 +20,12 @@ class HomePageBackButton extends StatelessWidget{
           actions: <Widget>[
             new RaisedButton(
                 onPressed: () => Navigator.of(context).pop(false),
-                child: new Text('No'),
+                child: new Text('Não'),
                 color: Theme.of(context).primaryColor,
             ),
             new RaisedButton(
                 onPressed: () => Navigator.of(context).pop(true),
-                child: new Text('Yes'),
+                child: new Text('Sim'),
```

Comment:
Translation really isn't the scope of this PR. It may stay here, but everything will be translated later on. Maybe even to english if the app is launched university wide